scavenger-agent-java/src/main/java/com/navercorp/scavenger/javaagent/collecting/ElementMatcherBuilder.java

```diff
@@ -55,11 +55,23 @@ public ElementMatcher<TypeDescription> buildClassMatcher() {
             .reduce(ElementMatcher.Junction::or)
             .orElse(none());
 
+        ElementMatcher.Junction<NamedElement> excludeByRegexMatcher = config.getExcludeByRegex().stream()
+            .map(ElementMatchers::nameMatches)
+            .reduce(ElementMatcher.Junction::or)
+            .orElse(none());
+
+        ElementMatcher.Junction<NamedElement> additionalByRegexMatcher = config.getExcludeByRegex().stream()
```

Comment:
```suggestion
        ElementMatcher.Junction<NamedElement> additionalByRegexMatcher = config.getAdditionalByRegex().stream()
```

There is a typo here.

Reply:
Oops.... fixed in 651d87debd035b4ab5499eb080271632d4cbfb87